pkgs/by-name/ka/karla/package.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation {
+  pname = "karla";
+  version = "2.004";
+
+  src = fetchFromGitHub {
+    owner = "googlefonts";
+    repo = "karla";
+    rev = "69b25f663101efb4113dd7ed416c120dd2dce56a";
+    hash = "sha256-9zf2gfhSn8Aly8a2CINsNXZPRfVcaxFE9In4Qqx3Fn4=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm444 -t $out/share/fonts/truetype fonts/ttf/*.ttf
+    install -Dm444 -t $out/share/fonts/variable fonts/variable/*.ttf
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "An outstanding grotesque sans serif typeface family";
```

ℹ️ According to section ["Meta attributes" in pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes)
`meta.description` should not start with a definite or indefinite article:
```suggestion
    description = "Outstanding grotesque sans serif typeface family";
```